index.html

```diff
@@ -228,6 +228,12 @@ <h1 class="hero-header">
           <p>
             If you need more information on voting or the election process, visit <a href="https://www.usa.gov/voting" target="blank">USA.gov/voting</a>. To find this information in Spanish, visit <a href="https://gobierno.usa.gov/elecciones-electorales" target="blank">GobiernoUSA.gov</a>.
           </p>
+          <p>
+            To learn more about our current progress &mdash; and to make suggestions &mdash;
+            visit our <a
+              href="https://github.com/presidential-innovation-fellows/vote-2016"
+              target="_blank">GitHub repository</a>.
```

Comment:
Ah yeah, thanks! ([Here's that issue.](https://github.com/presidential-innovation-fellows/apps-gov/issues/104)) I'd suggest "code repository and issue tracker" for this one, since you're pointing out the opportunity to make suggestions.